src/components/dev-dot-content/mdx-components/mdx-heading-permalink/mdx-heading-permalink.module.css

```diff
@@ -0,0 +1,42 @@
+.root {
+	composes: g-focus-ring-from-box-shadow from global;
+	opacity: 0;
+	border-radius: 6px;
+	padding: 4px;
+	height: fit-content;
+	float: right;
+	margin-left: 8px;
+
+	@media (prefers-reduced-motion: no-preference) {
+		transition: opacity 0.2s ease;
+	}
+
+	& .icon {
+		height: 16px;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+	}
+
+	&:hover {
+		opacity: 1;
+		box-shadow: var(--token-surface-mid-box-shadow);
+
+		& svg * {
+			fill: var(--token-color-foreground-action-hover);
+		}
+	}
+
+	&:focus {
+		opacity: 1;
+		background: var(--token-color-surface-faint);
+
+		& svg * {
+			fill: var(--token-color-foreground-action-active);
+		}
+	}
```

Comment:
I _think_ we should be able to set the colors of the `svg`s without the `*` character like this:

```css
& svg {
  color: <color>;
}
```

I'm pretty sure it's how we do it in most other places. Have you already tried this approach?

Reply:
You're right! Nope I didn't think of that.